src/README.md

```diff
@@ -0,0 +1,23 @@
+
+# Frontend Builder Common
+
+The repo responsible for building all of the frontends on cloud.redhat.com
+
+## Akamai Cache Buster
+
+This script is run automatically from Jenkins each time a frontend is deployed
+to `Prod`. It clears out all of the old cached versions of the application to make
+sure users are served up to date content
+
+### To Run
+
+```bash
+python bustCache.py /path/to/your/.edgerc nameOfApp
```

Comment:
It probably doesn't matter at all, but does `appName` in place of `nameOfApp` sounds better?

Reply:
Yes